docs/operators-and-hooks-ref.rst

```diff
@@ -250,6 +250,11 @@ Foundation.
      -
      - :mod:`airflow.providers.apache.hive.transfers.mysql_to_hive`
 
+   * - `MySQL <https://www.mysql.com/>`__
```

Comment:
I'm afraid none of these services are managed by Apache Foundation. However, this table is in the Apache section. Could you look for a better table?

Reply:
Already fixed